cumulusci/robotframework/CumulusCI.py

```diff
@@ -245,14 +245,29 @@ def debug(self):
         set_pdb_trace()
 
     def start_perf_timer(self):
-        """Start an elapsed time stopwatch for performance tests"""
+        """Start an elapsed time stopwatch for performance tests.
```

Comment:
It might be good to mention in the docs that `end perf timer` should be called to save the time. And/or, you might want to consider renaming `End Perf Timer` to `Stop Perf Timer` so that the start and stop keywords appear next to each other in the generated keyword documentation.

Reply:
I did rename them but also I think that the example is pretty clear.